alpha/pom.xml

```diff
@@ -35,24 +35,21 @@
     <module>alpha-server</module>
   </modules>
 
+  <properties>
+    <kamon.version>0.6.0</kamon.version>
+    <akka.version>2.3.12</akka.version>
+  </properties>
+
   <dependencies>
-    <dependency>
+   <dependency>
       <groupId>io.kamon</groupId>
-      <artifactId>kamon-core_2.12</artifactId>
+      <artifactId>kamon-core_2.11</artifactId>
```

Comment:
Why do you change the scala version from 2.12 to 2.11?

Reply:
kamon-autoweave  dependency 0.6.0 not available for 2.12 version. So i made all kamon dependency to 2.11 version so that all kamon classes should be in single version.

Reply:
I will modify to latest version which is 2.12 only